### ISSUE TITLE
Unable to start rails console

### ISSUE THREAD

**Issue by darkBuddha** [rails/spring#668]
- Ruby 2.7.5, can't upgrade due to incompatible [PayPal SDK](https://github.com/paypal/paypalhttp_ruby/issues/12). 
- Rails 7.0.0

Now i can't enter the rails console, neither in production nor development:
```
/home/x/.rvm/gems/ruby-2.7.5/gems/spring-2.1.1/lib/spring/application.rb:103:in `block in preload': undefined method `mechanism=' for ActiveSupport::Dependencies:Module (NoMethodError)
```

Any idea? Thanks.

**Comment by catacs** [comment]
Hey @darkBuddha 👋 
I have the same issue (with ruby 3.0.3 and rails 7.0.1). 
How did you solved this issue?
Thanks 🙏

**Comment by darkBuddha** [comment]
> Hey @darkBuddha 👋 
> 
> I have the same issue (with ruby 3.0.3 and rails 7.0.1). 
> 
> How did you solved this issue?
> 
> Thanks 🙏 

reverted back to 2.7.5... good luck!

**Comment by darkBuddha** [comment]
> Hey @darkBuddha 👋 I have the same issue (with ruby 3.0.3 and rails 7.0.1). How did you solved this issue? Thanks 🙏

reverted to Rails 6.1.4.4

**Comment by catacs** [comment]
Just played a bit with spring versions.

Some context:
I used these 2 gems for spring:
```
  gem 'spring'
  gem 'spring-watcher-listen', '~> 2.0.0'
```
And it generates this configuration in Gemfile.lock:
```
    spring (2.1.1)
    spring-watcher-listen (2.0.1)
      listen (>= 2.7, < 4.0)
      spring (>= 1.2, < 3.0)
```
That means `spring-watcher-listen` its limiting the spring version to `2.1.1. (last version is `4.0.0`). I decided to remove `spring-watcher-listen` and upgrade the spring version to latest one.

Then run `bundle exec spring binstub --all ` to update the files. and finally _voilà_, looks like now the project works.
